test/mocks/network/io_handle.h

```diff
@@ -32,6 +32,7 @@ class MockIoHandle : public IoHandle {
   MOCK_METHOD(bool, supportsUdpGro, (), (const));
   MOCK_METHOD(Api::SysCallIntResult, bind, (Address::InstanceConstSharedPtr address));
   MOCK_METHOD(Api::SysCallIntResult, listen, (int backlog));
+  MOCK_METHOD(IoHandlePtr, accept, (struct sockaddr * addr, socklen_t* addrlen, int flags));
```

Comment:
Is there some additional testing that we should add for your changes?  Are there specific reasons why additional testing is not needed?

Reply:
Up until now, this accept was hidden within libevent. So, whatever we tested until now should still be tested.